src/components/LogIn/LogIn.test.tsx

```diff
@@ -33,7 +33,6 @@ describe("LogIn", () => {
     });
     renderComponent(<LogIn />, {
       state,
-      path: "*",
```

Comment:
Because of the wrapping component in the renderComponent util some of these paths had to change.